src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ISymbolExtensions.cs

```diff
@@ -656,6 +656,77 @@ private static bool VerifyGetAwaiter(IMethodSymbol getAwaiter)
             return methods.Any(m => m.Name == WellKnownMemberNames.GetResult && !m.Parameters.Any());
         }
 
+        public static bool IsValidGetEnumerator(this IMethodSymbol symbol)
+            => symbol.Name == WellKnownMemberNames.GetEnumeratorMethodName &&
+            VerifyGetEnumerator(symbol);
```

Comment:
Align Verify with symbol

Reply:
That's inconsistent with the style above in the document:

https://github.com/dotnet/roslyn/blob/b0787c0654d8fa40a9a5ed6a1a218106c07de72b/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ISymbolExtensions.cs#L623-L625

Should I change them both, or leave it.